stdlib/public/core/StringBridge.swift

```diff
@@ -306,6 +301,30 @@ public final class _NSContiguousString : _SwiftNativeNSString {
     return try body(selfAsPointer, rhsAsPointer)
   }
 
+  @objc
+  var fastestEncoding: UInt {
+    return _core.elementWidth == 1 ? UInt(1) : UInt(10)
```

Comment:
Ideally this class should live in the Foundation overlay; however I am not certain if that can be done - but it does seem reasonable to move the SPI access to the Foundation overlay since it is the Foundation team's responsibility to maintain that working as an interoperability point.

I have more performance extensions that will need to be made in this area: specifically the other direction of the bridge is slow as well (which will need these constants too).